src/sentry/south_migrations/0513_auto__add_field_eventattachment_group.py

```diff
@@ -0,0 +1,1513 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
Will let @mattrobenolt  decide if this is supposed to be dangerous.

Reply:
This is a beefy table, I didn't anticipate there to be so many rows. I think I'd rather error on the dangerous side and we might have to coordinate stopping workers to apply it.

Reply:
And with it being a foreign key, it means we also need an index and constraint, so as-is, this will lock up the table. We'll need to figure out how to apply it even manually.

Reply:
This can probably just be a BoundedBigIntegerField if that will make things easier

Reply:
Oh, right. Yes. We need to do that anyways since we wanna split off the ForeignKey anyways. Then yes, let’s do that, and you’ll also need an explicit index on this field then as well. Unless you never need it for lookups.

Reply:
Only deletions... I guess we still want the index though?